src/assets/css/sections/benefits.css

```diff
@@ -0,0 +1,19 @@
+.benefits-section {
+    position: relative;
+    top: -218px;
+    left: 7.4%;
+}
+
+.benefits-section .benefits-header {
+    color: #000;
+    padding-top: 93px;
+    padding-left: 99px;
+}
+
+.benefits-section .benefits-content {
+    display: flex;
+    width: 88%;
+    justify-content: space-between;
+    margin-top: 120px;   
```

Comment:
padding, margin very large, please use flex instead don't hard code like here

Reply:
when I code, I thought I can use justify-content property with value space-around to align elements. But I saw that the distance is not evenly. So I use margin and padding. Can you give me some suggestions?